conf/messages.cy

```diff
@@ -304,7 +306,8 @@ pages.date-of-marriage.h2=Pryd y gwnaethoch briodi neu ffurfio partneriaeth sifi
 # How it works
 #********************************************************************************
 pages.how-it-works.heading=Sut y mae’n gweithio
-pages.how-it-works.lede-pre1=Mae <a href="https://www.gov.uk/marriage-allowance-guide">Lwfans Priodasol</a> yn caniat&#225;u i chi drosglwyddo £{0} o’ch Lwfans Personol i’ch priod neu bartner sifil.
+pages.how-it-works.lede-pre1=yn caniatáu i chi drosglwyddo £{0} o’ch Lwfans Personol i’ch priod neu bartner sifil.
+pages.how-it-works.lede-pre1.link-text=Mae Lwfans Priodasol
```

Comment:
The link text has slightly changed since the last message. Is  this expected? (Previously, the link did not include "Mae")